src/web/components/MainContentIndicator.tsx

```diff
@@ -0,0 +1,4 @@
+export const MainContentIndicator = () => (
+	// eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+	<div id="maincontent" tabIndex={0} />
```

Comment:
Would it be worth licking through to the 'h1' or something like that on the page? We'd have to ensure that every layout type had an h1 with an ID but then the user would immediately jump to an h1 and if it was a screen reader, it would read it out to them?

Reply:
>Not 100% sure on the reason for a seperate main content element to jump to but otherwise looks clean

I agree -I copied the implementation as it exists on frontend but it possibly could be improved? It's worth point out that 'main content' in this instance means where the body starts, so skipping past the title (H1) and the author and meta info (in addition to the menus and above the fold ads)

Reply:
Hm interesting yeah. `ArticleBody` probably makes sense then, yeah?